Dockerfile.benchmark

```diff
@@ -0,0 +1,6 @@
+FROM ruby:2.5
```

Comment:
Just a note: Our `.ruby-version` for this project is `2.4.1`, while the dockerfile for the benchmark specifies `2.5`
I'm assuming this is not a big issue, and this is probably best solved in a separate PR if the solution is to update `.rubyversion`, but i think it'd be good to make these two consistent.

Reply:
Yep, worth bumping them to match.